services/datamanager/builtin/builtin.go

```diff
@@ -308,7 +308,7 @@ func (svc *builtIn) initializeOrUpdateCollector(
 }
 
 // getCollectorFromConfig returns the collector and metadata that is referenced based on specific config atrributes
-func (svc *builtIn) getCollectorFromConfig(attributes dataCaptureConfig) (data.Collector, *componentMethodMetadata, error) {
+func (svc *builtIn) getCollectorFromConfig(attributes dataCaptureConfig) (data.Collector, *componentMethodMetadata, bool) {
```

Comment:
Is this bool needed? It seems to be be true if the returned results are non-nil and false if they are nil, so it seems possibly unnecessary

Reply:
Good point, changed